src/components/NetworkAssetSelectInput/NetworkAssetSelectInput.tsx

```diff
@@ -462,13 +467,16 @@ const NetworkAssetSelectInput = ({
         </LargeSelectedOption>
       )}
       {showSelectModal && preselectedNetwork && (
-        <SelectedOption onClick={(e) => {
-              e.stopPropagation()
-              setPreselectedNetwork(null)
-            }
-          }
-          disabled={disabled}>
-          {!!preselectedNetwork?.iconUrl && <RoundedImage url={preselectedNetwork?.iconUrl} title={preselectedNetwork.title} size={24} />}
+        <SelectedOption
+          onClick={(e) => {
+            e.stopPropagation();
```

Comment:
it's a div, no need for `e.stopPropagation();`

Reply:
this is an existing function call, and I guess div does propagate event.